lib/TableGen/Operations.cpp

```diff
@@ -570,24 +626,48 @@ void BuilderMethod::emitDefinition(raw_ostream &out, FmtContext &fmt,
     }
 
     fmt.addSubst("args", symbols.chooseName({"args", "arguments"}));
-    out << tgfmt("::llvm::Value* const $args[] = {\n", &fmt);
+    out << tgfmt("std::vector<::llvm::Value*> $args = {\n", &fmt);
```

Comment:
Maybe use a SmallVector and use the number of arguments (1 for varargs) as the stack size?